packages/sdk/src/tests/multi_ne/ephemeralEvents.test.ts

```diff
@@ -139,7 +139,12 @@ describe('ephemeralEvents', () => {
         await expect(chuckEventDecryptedPromise).resolves.not.toThrow()
 
         expect(ephemeralSolicitations).toEqual([true])
-        await waitFor(() => expect(ephemeralFulfillments.length).toEqual(1))
-        expect(ephemeralFulfillments).toEqual([true])
+        
+        // Wait for at least one ephemeral fulfillment
+        await waitFor(() => expect(ephemeralFulfillments.length).toBeGreaterThanOrEqual(1))
+        
+        // Verify all fulfillments are from different clients
+        const uniqueSenders = new Set(ephemeralFulfillments)
```

Comment:
sets in typescript will compare objects by reference, so this will always be true, you need to convert the user address to a string and make a set of that.